extensions/big-data-cluster/src/bigDataCluster/dialog/mountHdfsDialog.ts

```diff
@@ -0,0 +1,367 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+'use strict';
+
+import * as vscode from 'vscode';
+import * as azdata from 'azdata';
+import * as nls from 'vscode-nls';
+import { ClusterController, ControllerError } from '../controller/clusterControllerApi';
+import { AuthType } from '../constants';
+
+const localize = nls.loadMessageBundle();
+
+const basicAuthDisplay = localize('basicAuthName', "Basic");
+const integratedAuthDisplay = localize('integratedAuthName', "Windows Authentication");
+
+function getAuthCategory(name: AuthType): azdata.CategoryValue {
+	if (name === 'basic') {
+		return { name: name, displayName: basicAuthDisplay };
+	}
+	return { name: name, displayName: integratedAuthDisplay };
+}
+
+/**
+ * Converts a comma-delimited set of key value pair credentials to a JSON object.
+ * This code is taken from the azdata implementation written in Python
+ */
+function convertCredsToJson(creds: string): {} {
```

Comment:
Its better for the return to have a more verbose output, for example `{ string: {} }` based on https://github.com/microsoft/azuredatastudio/pull/7580/files#diff-ca2c913185f9f071329a23c35ab59573R35